app/app.js

```diff
@@ -78,8 +78,13 @@ const render = (messages) => {
   );
```

Comment:
i know it's a bit outside the scope of this PR, but this has been bothering me for a while... i feel like explicitly unmounting component in hmr callback is more appropriate in this case, ie -

```js
 const MOUNT_NODE = document.getElementById('app')

// ...

 const render = (messages) => {
   ReactDOM.render(
     <Provider store={store}>
       <LanguageProvider messages={messages}>
         <ConnectedRouter history={history}>
           <App />
         </ConnectedRouter>
       </LanguageProvider>
     </Provider>,
     MOUNT_NODE
    )

// ...

  module.hot.accept('containers/App', () => {
    ReactDOM.unmountComponentAtNode(MOUNT_NODE)
    render(translationMessages);
  });
```